app/assets/javascripts/components/dashboard/navigation/panels/signals/signals_pane.jsx

```diff
@@ -1,42 +1,46 @@
-var SignalsPane = React.createClass({
-  render: function() {
-    return (
-      <div>
-        <h4 className='signal-header'>{this.renderPanelTitle()}</h4>
-        {this.choosePanel()}
-        <CreateNew 
-          handleClick={this.props.handleClick} 
-          handleSignal={this.props.handleSignal}
-        />
-      </div>
-    );
-  },
-
-  renderPanelTitle: function() {
-    var signalCount = this.props.signals.length;
+import React, { Component } from 'react';
+import WelcomePanel from './welcome_panel.jsx';
+import ActiveSignalPanel from './active_signal_panel.jsx';
+import CreateNew from './create_new.jsx';
 
+export default class SignalsPane extends Component {
+  renderPanelTitle() {
+    const signalCount = this.props.signals.length
     if (signalCount == 0) {
       return 'All Signals'
     } else if (signalCount == 1) {
       return 'Signal - 1 Active'
     } else {
       return 'Signals - ' + signalCount + ' Active'
     }
-  },
+  }
 
-  choosePanel: function() {
-    var signalCount = this.props.signals.length
+  choosePanel() {
+    const signalCount = this.props.signals.length
```

Comment:
Try not to use `var`s. Prefer `let` or `const`. `const` is used when you will not redefine the variable and `let` is used when you _will_ be redefining it.